aurora/security_group.tf

```diff
@@ -20,6 +20,7 @@ resource "aws_security_group" "sg_aurora" {
 
   lifecycle {
     create_before_destroy = true
+    ignore_changes        = [name]
```

Comment:
Does this really trigger a destroy/recreate otherwise? AFAIK `name` on security groups is just a changeable tag.

Reply:
Nope, it does cause a destroy 

```
+/- resource "aws_security_group" "sg_rds" {
      ~ arn                    = "arn:aws:ec2:eu-west-1:213294516265:security-group/sg-0e594ed9d28da45e2" -> (known after apply)
        description            = "Security group that is needed for the RDS"
      ~ egress                 = [] -> (known after apply)
      ~ id                     = "sg-0e594ed9d28da45e2" -> (known after apply)
      ~ ingress                = [
          - {
              - cidr_blocks      = []
              - description      = ""
              - from_port        = 3306
              - ipv6_cidr_blocks = []
              - prefix_list_ids  = []
              - protocol         = "tcp"
              - security_groups  = [
                  - "sg-14bde66d",
                  - "sg-64bce71d",
                  - "sg-bf55f6c7",
                  - "sg-c3bde6ba",
                  - "sg-d2748aaa",
                  - "sg-e4b9e29d",
                ]
              - self             = false
              - to_port          = 3306
            },
        ] -> (known after apply)
      ~ name                   = "sweepbright-dev-rds" -> "sweepbright-dev" **# forces replacement**
      ~ owner_id               = "213294516265" -> (known after apply)
        revoke_rules_on_delete = false
      ~ tags                   = {
            "Environment" = "dev"
          ~ "Name"        = "sweepbright-dev-rds" -> "sweepbright-dev"
            "Project"     = "sweepbright"
        }
        vpc_id                 = "vpc-fc8b4f9b"

      - timeouts {}
    }
```